### PR TITLE
Angular homepage

### PR DESCRIPTION
**Pull request by kwalrath**
Staged version: https://kw-webdev-dartlang-1.firebaseapp.com/angular

You might need to clear the cache and/or open it in an incognito window to avoid redirecting to /angular/guide.

### CODE REVIEW

File: src/angular/index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+layout: angular
+title: About AngularDart
+description: Angular + Dart = productivity, performance, and stability
+toc: true
```
Comment (chalin) [inline src/angular/index.md]: I'd suggest `toc: false`; the page is so short (even in a narrow view), I'm not sure that adding a(n autogenerated) TOC has much value.

File: src/_data/side-nav.yml [hunk 2]
```diff
@@ -3,10 +3,10 @@
   - title: Get Started
     permalink: /guides/get-started
 
-- title: Angular Docs
+- title: Angular
   children:
-    - title: Quickstart
-      permalink: /angular/quickstart
+    - title: About AngularDart
```
Comment (chalin) [inline src/_data/side-nav.yml]: How about just "About" in the sidenav?

File: src/angular/index.md (hunk 1 above)
Comment (kwalrath) [inline src/angular/index.md]: Yep, that was an oversight.

File: src/_data/side-nav.yml (hunk 2 above)
Comment (kwalrath) [inline src/_data/side-nav.yml]: I like it!

**Comment by kwalrath** [comment]
I'm inclined to ship this. We can iterate, of course.

**Pull request merged**